spec/models/phone_number_spec.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe PhoneNumber, type: :model do
+  describe 'validations' do
+    let(:user) do
+      User.create(username: 'Mike',
+                  biography: 'I am a developer',
+                  age: 25)
+    end
+
+    let(:contact) do
+      Contact.create(name: 'Rudolph',
+                     user_id: user.id)
+    end
+
+    subject do
+      PhoneNumber.new(phone_number: '00123456789',
+                      label: 'work',
+                      primary: FalseClass,
+                      contact_id: contact.id)
+    end
+
+    context 'when all attributes are present' do
+      it 'is valid' do
+        expect(subject.valid?).to be_truthy
+      end
+    end
+
+    context 'when number is not present' do
+      it 'is not valid without a phone_number' do
+        subject.phone_number = nil
```

Comment:
Para ficar melhor organizado, a parte que você altera o item, coloque fora do it, veja formas de fazer isso, com let ou before por exemplo. O it fica somente com o expect.